### PR TITLE
Speed up writecache initialization

### PR DESCRIPTION
**Pull request by fyrchik**

### CODE REVIEW

**Review by acid-ant — APPROVED** [review]

File: pkg/local_object_storage/writecache/init.go [hunk 1]
```diff
@@ -46,29 +59,53 @@ func (c *cache) initFlushMarks() {
...
 		lastKey = append([]byte(m[len(m)-1]), 0)
 	}
 
 	c.log.Info("finished updating flush marks")
 }
 
-func (c *cache) isFlushed(addr oid.Address) bool {
-	var prm meta.StorageIDPrm
-	prm.SetAddress(addr)
+func (c *cache) flushStatus(addr oid.Address) (bool, bool) {
```
Comment (carpawell) [inline pkg/local_object_storage/writecache/init.go:95]: add docs about duplicated returned types?

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing. Build number is 3722

**Comment by sami-nspcc** [comment]
I am running integration tests

**Comment by sami-nspcc** [comment]
Test run is finished. Please download the tarball from <a href="https://http.t5.fs.neo.org/get/FszwwAx3VznDysQGxBUMRj1zj3zp8VcohorVDnunrviV/37oecp3n5Jy2HQfteJtS6Sjnj1Q3WeWSGKR92FUxjNb9">link</a>. Untar and use `allure open` to watch the report

**Review by acid-ant — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Pull request merged**